sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -971,6 +971,20 @@ def _add_argparse_args(cls, parser):
             'override. Values can be either a labeled level or a number '
             '(See https://docs.python.org/3/library/logging.html#levels). '
             'Default log level is INFO.'))
+    parser.add_argument(
+        '--sdk_harness_log_level_overrides',
+        action='append',
+        default=None,
+        help=(
+            'Controls the log levels for specifically named loggers. The '
+            'expected format is a json string: {\"module\":\"log_level\",...}. '
+            'For example, by specifying the value "{\"a.b.c\":\"DEBUG\"}, '
+            'the logger underneath the module "a.b.c" will be configured to '
+            'output logs at the DEBUG level. Similarly, by specifying the '
+            'value {"a.b.c":"WARNING"} all loggers underneath the "a.b.c" '
```

Comment:
is escaping quotes necessary here as well?

Reply:
yeah, not necessary. Copied from java sdk and found I even missed a quote mark. Will fix